src/snowflake/connector/pandas_tools.py

```diff
@@ -54,7 +52,7 @@ def write_pandas(
     compression: str = "gzip",
     on_error: str = "abort_statement",
     parallel: int = 4,
-    **kwargs
+    quote_identifiers: bool = True,
```

Comment:
LGTM. Two doc related suggestions: 1, the summary of this PR has to update accordingly - since it is no more kwargs. 2) A short description about the problem and when to use quote_identifier with what value for this parameter should be included either in document or in comments here.